packages/cozy-harvest-lib/src/components/KonnectorUpdateLinker.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import { Query, withClient } from 'cozy-client'
+import AppLinker from 'cozy-ui/react/AppLinker'
+import Button from 'cozy-ui/react/Button'
+
+const STORE_SLUG = 'store'
+
+const getKonnectorUpdateUrl = (appData = [], konnector = {}) => {
+  const storeApp = appData.find(
+    app => app.attributes && app.attributes.slug === STORE_SLUG
+  )
+  if (!storeApp) return null
+
+  const storeUrl = storeApp.links && storeApp.links.related
+  if (!storeUrl) return null
+
+  return `${storeUrl}#/discover/${konnector.slug}/install`
+}
+
+const KonnectorUpdateButton = ({
+  disabled,
+  isBlocking,
+  href,
+  label,
+  onClick
+}) => (
+  <Button
+    disabled={disabled}
+    className="u-m-0"
+    href={href}
+    icon="eye"
+    label={label}
+    onClick={onClick}
+    theme={isBlocking ? 'danger' : 'secondary'}
+  />
+)
+
+export class KonnectorUpdateLinker extends PureComponent {
+  render() {
+    const { client, label, isBlocking, konnector } = this.props
+    return (
+      <Query query={client.find('io.cozy.apps')}>
+        {({ data, fetchStatus }) => {
+          const isFetching = fetchStatus !== 'loaded'
```

Comment:
wrong. If fetchStatus === 'errored' we are not fetching and I want to display an error message instead of a loader